qemu/lib/iozone.py

```diff
@@ -0,0 +1,133 @@
+import logging
+import os
+
+from virttest import utils_test
+from virttest import utils_misc
+from virttest import postprocess_iozone
+
+
+class IozoneTest(object):
+    """
+    Prepare iozone env, run iozone on guest, clean up iozone env.
+    """
+    def __init__(self, vm, params, test):
+        """
+        run setup of iozone test
+        """
+        self.params = params
+        self.session = vm.wait_for_login()
+        self.test = test
+        self.results_path = ""
+        self.timeout = float(params.get("iozone_timeout", 240))
+
+    def before_run_iozone(self):
+        """
+        Prepare for iozone test
+        """
+        raise NotImplementedError()
+
+    def run_iozone_in_guest(self, disk_index, mountpoint):
+        """
+        Run iozone test in guest
+
+        :param disk_index: index to find disk under test
+        :param mountpoint: mountpoint/disk letter assigned to disk under test
+        :return: iozone run status and result.
+        """
+        iozone_test_cmd_default = "./iozone -azR -r 64k -n 125M -g 512M -i 0 -i 1 %s"
+        iozone_test_cmd = self.params.get("iozone_test_cmd", iozone_test_cmd_default)
+        if self.params.get("os_type") == "windows":
+            if self.params.get("format_disk", "yes") == "yes":
+                logging.info("Format disk")
+                utils_misc.format_windows_disk(self.session, disk_index, mountpoint)
+            test_cmd = iozone_test_cmd % (mountpoint, mountpoint)
```

Comment:
Hi, for windows guest:
1. when use the iozone_test_cmd_default, it only need one %s, but here have 2 % (mountpoint, mountpoint).
2. If do not use the iozone_test_cmd_default, as you test.cfg written, iozone_test_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 125M -g 512M -M"
iozone_test_cmd += " -b %s:\iozone_test -f %s:\testfile", the first %s should be different with the second %s, first is for log saved, second is for tested file, if the tested disk is damaged, the log will not be saved, the first should be system disk letter, like "iozone -az -b c:\iozone_test -g 4g -y 32k -i 0 -i 1 -f I:\test-data".